app/container.php

```diff
@@ -40,12 +34,40 @@
 
 $container = new Container();
 
+$osDetector = new OperatingSystemDetector();
+$operatingSystem = $osDetector->get();
```

Comment:
All the code bellow in the container can be simplified if the `OperatingSystemDetector` class has `isMac` and `isDebianBased` methods.